airbyte-workers/src/main/java/io/airbyte/workers/process/KubeProcessFactory.java

```diff
@@ -117,4 +117,38 @@ public Process create(String jobId,
     }
   }
 
+  /**
+   * Docker image names are by convention separated by slashes. The last portion is the image's name.
+   * This is followed by a colon and a version number. e.g. airbyte/scheduler:v1 or
+   * gcr.io/my-project/image-name:v2.
+   *
+   * Kubernetes has a maximum pod name length of 63 characters.
+   *
+   * With these two facts, attempt to construct a unique Pod name with the image name present for
+   * easier operations.
+   */
+  @VisibleForTesting
+  protected static String getPodName(String fullImagePath, String jobId, int attempt) {
```

Comment:
maybe `createPodName`? 

This is a great idea btw.

Reply:
done!